.yaydoc.yml

```diff
@@ -0,0 +1,35 @@
+metadata:
+  author: FOSSASIA
+  projectname: "Susi Server"
+  version: development
+build:
+  theme:
+    name: sphinx_fossasia_theme   
+  source: docs
+  logo: images/susi-logo-original.svg
+  subproject:
+    - url: https://github.com/fossasia/susi_android
+      source: docs
```

Comment:
@imujjwal96 I suggested it because seeing `source: docs` repeated again and again didn't looked good. Also I don't think the `docs` directory would be renamed soon. Although, if you think the yaml file should be explicit here, I'd be ok with that.